app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -149,6 +150,10 @@ public void onCreate(Bundle savedInstanceState) {
         } else {
             loginCredentials.setVisibility(View.GONE);
         }
+
+        getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_ADJUST_PAN);
+
+
```

Comment:
This is unrelated, right?

Reply:
Hi @nicolas-raoul I branched from my old master branch which has an merged code  is it okay if you can merge those changes so that i can make a new fork